packages/ozone/src/mod-service/lang.ts

```diff
@@ -40,6 +45,27 @@ export class ModerationLangService {
     }
   }
 
+  getTextFromRecord(
+    uri: string,
+    recordValue?: Record<string, unknown>,
+  ): string | undefined {
+    const atUri = new AtUri(uri)
+    let text: string | undefined
+    if (
+      [ids.AppBskyGraphList, ids.AppBskyFeedGenerator].includes(
+        atUri.collection,
+      )
+    ) {
+      text = (recordValue?.description || recordValue?.name) as string
+    }
+
+    if (!text && recordValue?.text) {
+      text = recordValue.text as string
+    }
+
+    return text?.trim()
```

Comment:
hmm... I took a few smaples from prod and they are always strings so thought this was safe. what would you recommend here. we could always wrap this in a try catch and return falsy value so that the entire evaluation doesn't fail. keep in mind this is meant to be a fallback and eventually, we should get the right lang tags from the record itself instead of having to determine from the text.